{"ID": "4bbba5cd-6f46-4077-8012-ed1a0a67d429", "effective_cve": "CVE-2021-31810", "image": {"exact": "docker.io/anchore/test_images@sha256:808f6cf3cf4473eb39ff9bb47ead639d2ed71255b75b9b140162b58c6102bcc9"}, "label": "FP", "package": {"name": "ruby", "version": "2.5.9-110.module_el8.6.0+1187+541216eb"}, "timestamp": "2022-11-01T16:10:23+00:00", "tool": "grype@v0.51.0-7-gfcce63b", "user": "westonsteimel", "vulnerability_id": "CVE-2021-31810"}